app/assets/stylesheets/govuk_publishing_components/components/_contextual-sidebar.scss

```diff
@@ -38,6 +36,8 @@ $transition-campaign-dark-blue: #1e1348;
 }
 
 .gem-c-contextual-sidebar__brexit-cta:focus {
+  box-shadow: 0 $govuk-focus-width $govuk-focus-text-colour;
```

Comment:
You can do `@include govuk-focused-text;` although I'm not 100% sure that would work here, just mentioning for reference in case it's useful.

Reply:
I haven't got around to adding screenshots to show the difference as this is still in draft. I'm almost doing the same as `govuk-focused-text` - this doesn't add the the slight yellow border at the top that appears above the red border.